### PR TITLE
Added calc_table_metrics

### LINKED ISSUE #5: Set up and run experiment one
In some ways this is going to be the experiment that let's us verify that that the calibration workflow is working as expected. This will be re-implementing the calibration of beta, q10, and diff that was used in the V3 manuscript.

### LINKED ISSUE #14: Add table metrics functions
Integrate table metrics script and functions into major_functions.R in order to evaluate parameters output by optim.

### PR DESCRIPTION
**Pull request by ptrscll**
I added in the calc_table_metrics and tested it (in my scratch file) to confirm it produces the same output as it did in script form in the old repository. This PR also includes adding the idealized runs folder so that all metrics can be calculated.

Note: I was originally planning on running experiment one on this PR (hence the branch name) but I think it might make more sense to make a separate PR for that.

### CODE REVIEW

**Comment by ptrscll** [comment]
Update: After starting work on running experiment one and realizing I had to make some changes to run_optim, I think it might make sense to keep using this PR for experiment one stuff

**Comment by ptrscll** [comment]
I got a calibration script to run, but it was calibrating to CO2 data that only goes back to 1959 (from Mauna Loa Observatory). Results of this initial run (including table metrics) are included in the results file.

File: results/initial_experiment.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+***Key Metrics***
```
Comment (kdorheim) [inline results/initial_experiment.txt:136]: Could you add the fitted parameter values  & the RMSE values to the tables?

**Review by kdorheim — CHANGES_REQUESTED** [review]

**Comment by ptrscll** [comment]
I just tried using the new CO2 data, but I seem to have gotten some different results from the original calibration run. Looking into it now

**Comment by ptrscll** [comment]
I was able to get my optimization procedure to match the Hector defaults/manuscript results! I also included some previous attempts in my initial_experiment.txt results file, since there were some interesting results in there. To summarize:

- Adding in the 1750 CO2 data point didn't substantially change my optimization results
- With Nelder-Mead (and no lower/upper bounds on parameters), I was able to achieve a lower averaged MSE of 2.30 instead of 2.33 with L-BFGS-B
- While using Nelder-Mead, when I changed my initial parameters, I saw a change in optimal ocean heat diffusivity from 1.61 to 3.40 despite negligible changes in either of the other two parameters or in my averaged MSE of 2.30
- Once I switched to using L-BFGS-B with the initial parameters/standard deviations from the manuscript code, I was able to reproduce the default parameters and results from the manuscript tables
- Although the difference is not nearly as large as with my incorrect error function, the vast majority of the averaged MSE in all of these runs is coming from CO2 (MSE of ~ 4.55) rather than from temperature (MSE of ~0.08). This may be why there was that weird behavior with ocean heat diffusivity?

**Pull request merged**